packages/server-wallet/src/message-service/socket-io-message-service.ts

```diff
@@ -1,12 +1,20 @@
 import {Server} from 'socket.io';
 import {io, Socket as ClientSocket} from 'socket.io-client';
 import {Message} from '@statechannels/client-api-schema';
+import delay from 'delay';
+import {AbortController} from 'abort-controller';
 
 import {MessageHandler, MessageServiceFactory, MessageServiceInterface} from './types';
+import {LatencyOptions} from './test-message-service';
 
 export class SocketIOMessageService implements MessageServiceInterface {
   private server: Server;
   private peers: ClientSocket[] = [];
+  private _frozen = false;
```

Comment:
Can this variable ever get toggled? Have we tested this option?

Reply:
Oops, this was copy and pasted by accident.

Reply:
Removed in 4d37d37